src/test/java/com/amazon/ion/impl/macro/MacroEvaluatorTest.kt

```diff
@@ -560,9 +601,13 @@ class MacroEvaluatorTest {
             }
         }
 
+        assertIsInstance<StructValue>(evaluator.expandNext())
```

Comment:
🗺️ This test for `make_field` was wrong. It was testing that the macro produced a field name and a value, but should have been testing that it produced a struct containing a single name-value pair.